pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -1061,7 +1044,8 @@ protected boolean isExceedMaximumMessageSize(int size, PublishContext publishCon
     /**
      * update topic publish dispatcher for this topic.
      */
-    protected void updatePublishDispatcher(PublishRate publishRate) {
+    public void updatePublishDispatcher() {
+        PublishRate publishRate = topicPolicies.getPublishRate().get();
```

Comment:
Can this return null?

Reply:
It can not return null, I will remove the `if (publishRate != null` below

Reply:
@eolivelli  PTAL